docs/integrations/sources/mysql.md

```diff
@@ -1,91 +1,55 @@
 
-| Feature                       | Supported | Notes                             |
-| :---------------------------- | :-------- | :-------------------------------- |
-| Full Refresh Sync             | Yes       |                                   |
-| Incremental - Append Sync     | Yes       |                                   |
-| Replicate Incremental Deletes | Yes       |                                   |
-| CDC                           | Yes       |                                   |
-| SSL Support                   | Yes       |                                   |
-| SSH Tunnel Connection         | Yes       |                                   |
-| Namespaces                    | Yes       | Enabled by default                |
-| Arrays                        | Yes       | Byte arrays are not supported yet |
+The contents below include a 'Quick Start' guide, advanced setup steps, and reference information (data type mapping, and changelogs).
 
-The MySQL source does not alter the schema present in your database. Depending on the destination connected to this source, however, the schema may be altered. See the destination's documentation for more details.
+![Airbyte MySQL Connection](https://raw.githubusercontent.com/airbytehq/airbyte/3a9264666b7b9b9d10ef8d174b8454a6c7e57560/docs/integrations/sources/mysql/assets/airbyte_mysql_source.png)
 
-## Troubleshooting
+## Quick Start
 
-There may be problems with mapping values in MySQL's datetime field to other relational data stores. MySQL permits zero values for date/time instead of NULL which may not be accepted by other data stores. To work around this problem, you can pass the following key value pair in the JDBC connector of the source setting `zerodatetimebehavior=Converttonull`.
+Here is an outline of the minimum required steps to configure a MySQL connector:
+1. Create a dedicated read-only MySQL user with permissions for replicating data
+2. Create a new MySQL source in the Airbyte UI using CDC logical replication
+3. (Airbyte Cloud Only) Allow inbound traffic from Airbyte IPs
 
-Some users reported that they could not connect to Amazon RDS MySQL or MariaDB. This can be diagnosed with the error message: `Cannot create a PoolableConnectionFactory`.
-To solve this issue add `enabledTLSProtocols=TLSv1.2` in the JDBC parameters.
+Once this is complete, you will be able to select MySQL as a source for replicating data.
 
-Another error that users have reported when trying to connect to Amazon RDS MySQL is `Error: HikariPool-1 - Connection is not available, request timed out after 30001ms.`. Many times this is can be due to the VPC not allowing public traffic, however, we recommend going through [this AWS troubleshooting checklist](https://aws.amazon.com/premiumsupport/knowledge-center/rds-cannot-connect/) to the correct permissions/settings have been granted to allow connection to your database.
+<FieldAnchor field="username,password">
 
```

Comment:
```suggestion
<FieldAnchor field="username,password">
```

I'm still investigating why this is the case, but the `#### Step 1:...` heading doesn't render properly in the Airbyte UI if there is an empty line between it and the <FieldAnchor> tag. 

I don't want to block this change on me investigating the bug so let's just get rid of the empty line here for now